src/component/archive/ArchiveTable/ArchiveTable.tsx

```diff
@@ -153,12 +158,16 @@ export const ArchiveTable = ({
                 Header: 'Actions',
                 id: 'Actions',
                 align: 'center',
-                maxWidth: 85,
+                maxWidth: 120,
                 canSort: false,
-                Cell: ({ row: { original } }: any) => (
-                    <ReviveArchivedFeatureCell
-                        project={original.project}
-                        onRevive={() => onRevive(original.name)}
+                Cell: ({ row: { feature } }: any) => (
```

Comment:
Seems like this change is currently breaking `/archive`.
Try destructuring the object like this instead:
```suggestion
                Cell: ({ row: { original: feature } }: any) => (
```

Reply:
Done